docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.2"
+
+services:
+  dev:
+    container_name: node-bittensor
+    image: "bittensor/bittensor:latest"
+    ports:
+      - "8091:8091"
+    
+    command: /bin/bash -c "
+      cd /bittensor 
+      && python3.7 examples/IMAGE/mnist.py --metagraph.chain_endpoint 172.17.0.1:9944"
```

Comment:
Does this point to feynman?

Reply:
No, this is the docker bridge network between the two containers running on the same computer.